packages/styled-system/src/css.ts

```diff
@@ -147,7 +147,15 @@ export const css = (args: StyleObjectOrFn = {}) => (
     const val = runIfFn(x, theme)
 
     const key = k in pseudoSelectors ? pseudoSelectors[k] : k
-    const config = (parser.config as Dict)[key]
+    let config = (parser.config as Dict)[key]
```

Comment:
What's happening if the config module is false?

Reply:
by design, the config module only takes a true value if it's meant to be translated to an config object. https://styled-system.com/custom-props#shortcut-definition
the intention for a false value needs to be defined. the current behavior for a false value is not parsing the token against the theme and output the value directly.

Reply:
by design, the config module only takes a true value if it's meant to be translated to an config object. styled-system.com/custom-props#shortcut-definition
the intention for a false value needs to be defined. the current behavior for a false value is not parsing the token against the theme and output the value directly.

Reply:
Approved.

Reply:
thanks!